packages/monorepo/lib/update-package-json.js

```diff
@@ -64,7 +64,12 @@ async function updatePackageJsonFiles(options) {
       directory: path.relative(p.rootPath, p.location).replace(/\\/g, '/'),
     };
 
-    pkg.engines = rootPkg.engines;
+    if ((pkg.private === true && Array.isArray(pkg.workspaces)) || pkg.name === '@loopback/tsdocs-monorepo') {
```

Comment:
I have seen that we can do even better. If you change the variable 'isLernaRepo' that is in line 44 to this
```ts
const isLernaRepo = (pkg.private === true && Array.isArray(pkg.workspaces)) || fs.existsSync(path.join(p.location, 'lerna.json')); // rename for example to isMonorepoPackage
```

You can do something like this
```suggestion
    if (isMonorepoPackage) {
```

Reply:
The package @loopback/tsdocs-monorepo goes without saying, it is already included by the lerna package

Reply:
Great suggestion; I've applied the changes.

Is there any reason for checking `pkg.private = true`? I think check the existence of `pkg.workspaces` should be suffice.

Reply:
Normally it is usually put like this but if you prefer to put it that way it is equally fine